grakn-engine/src/main/java/ai/grakn/engine/controller/SystemController.java

```diff
@@ -71,10 +76,45 @@
     private final Logger LOG = LoggerFactory.getLogger(SystemController.class);
     private final EngineGraknGraphFactory factory;
 
-    public SystemController(EngineGraknGraphFactory factory, Service spark) {
+    public SystemController(EngineGraknGraphFactory factory , Service spark) {
         this.factory = factory;
-        spark.get(KEYSPACES,     this::getKeyspaces);
-        spark.get(CONFIGURATION, this::getConfiguration);
+
+        spark.get(INITIALISE,         this::initialiseSession);
```

Comment:
Sometimes I query for the configuration using the URL just to check its contents. Figured it was worth maintaining for that reason?